_includes/selector.html

```diff
@@ -744,11 +746,10 @@
 
                     This handles that case, updating to the oldest Python version supported by the chosen release.
                 */
-                if (this.active_release === "Stable" && !this.python_vers_stable.includes(this.active_python_ver)) {
-                    this.active_python_ver = this.python_vers_stable[0];
-                } else if (this.active_release === "Nightly" && !this.python_vers_nightly.includes(this.active_python_ver)) {
-                    this.active_python_ver = this.python_vers_nightly[0];
-                }
+               var supported_python_versions = this.getSupportedPythonVersions()
+               if (!supported_python_versions.includes(this.active_python_ver)) {
+                   this.active_python_ver = supported_python_versions[0];
```

Comment:
Prefer the latest Python version.
```suggestion
                   this.active_python_ver = supported_python_versions[supported_python_versions.length - 1];
```

Reply:
sure, fine with me.